Update 03_modify_and_scale_cluster.mdx (#1656)
EnterpriseDB/docs: product_docs/docs/edbcloud/beta/using_cluster/03_modify_and_scale_cluster.mdx

title: "Modifying and Scaling Clusters"
---

> 1. From the Clusters page, click on the name of the cluster you want to edit.

Reviewer: ```suggestion
1. From the **Clusters** page, select the name of the cluster you want to edit.
```

> 2. Click Edit Cluster from the top right corner of the Cluster Info panel.

Reviewer: ```suggestion
2. Select **Edit Cluster** from the top right corner of the **Cluster Info** panel.
```

> 5. Click Save to save your changes.

Reviewer: ```suggestion
5. Select **Save** to save your changes.
```

> 6. Click Confirm Changes. !!! Note: Saving changes may require a database restart.

Reviewer: ```suggestion
6. Select **Confirm Changes**. !!! Note: Saving changes may require a database restart.
```
